Sources/ZcashLightClientKit/Synchronizer.swift

```diff
@@ -309,8 +309,11 @@ public protocol Synchronizer: AnyObject {
     /// - Returns: `AccountBalance`, struct that holds sapling and unshielded balances or `nil` when no account is associated with `accountIndex`
     func getAccountBalance(accountIndex: Int) async throws -> AccountBalance?
 
+    /// Returns the combine's subject for exchange rate, when subscribed to it, the value is returned.
```

Comment:
What does "the combine's subject" mean here? `Synchronizer` is usable independently of `CombineSynchronizer` AFAIK, and we also have `ClosureSynchronizer`.

I think this should read something more like:
```suggestion
    /// The latest known USD/ZEC exchange rate, paired with the time it was queried.
    ///
    /// The rate can be initialized and refreshed by calling `refreshExchangeRateUSD`.
```
(with additional details if necessary about being able to subscribe to this, if that's how this API works).